dbos/_core.py

```diff
@@ -662,17 +700,22 @@ def wrapper(*args: Any, **kwargs: Any) -> R:
 
         wfOutcome = Outcome[R].make(functools.partial(func, *args, **kwargs))
 
+        workflow_id = None
```

Comment:
I think `workflow_id` could be confusing -- can we call it `child_workflow_id` to be more specific?

Reply:
It is the workflow ID&mdash;this is the right name.